components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/codegen/CodeGenerator.java

```diff
@@ -110,7 +111,7 @@ public void generate(String projectName, List<ExtendedAPI> apis, boolean overwri
      * @throws IOException                  when file operations fail
      * @throws BallerinaServiceGenException when code generator fails
      */
-    public void generate(String projectName, String[] apiDefinitions, String[] endpointDefinitions, boolean overwrite)
+    public void generate(String projectName, String[] apiDefinitions, String[] endpointDefinitions, boolean[] defaultAPI, boolean overwrite)
```

Comment:
`defaultAPI` name doesn't describe the variable as it should be. Change it to something meaningful

Ex:
```suggestion
    public void generate(String projectName, String[] apiDefinitions, String[] endpointDefinitions, boolean[] defaultAPIs, boolean overwrite)
```